src/install/java/agentConfig.mdx

```diff
@@ -0,0 +1,19 @@
+---
+componentType: agentConfig
+headingText: Choose your download option
+descriptionText: Set your app name and license key. Then download your custom config file. You can also use more advanced config setting, see our docs.
+inputOptions: 
+  - labelMdx: 'Name your app'
+    value: appName
+    infoMdx: 'some random info text' 
+  - label: 'Enter your [new relic license key]()'
+    value: licenseKey
+    infoMdx: 'some random info text' 
```

Comment:
what do you think about changing this to line up a bit more with the component?
```suggestion
  - label: 'Name your app'
    name: appName
    codeLine: 31
    defaultValue: 'My Application'
    toolTip: 'some random info text' 
  - label: 'Enter your '
    url: 
      href: '/'
      title: 'New Relic license key'
    name: licenseKey
    codeLine: 15
    defaultValue: '12345'
```

Reply:
yes! I can do that, let me make those changes to the schema